bibblio.py

```diff
@@ -213,6 +327,21 @@ def output_source(self):
         return DataSource.lookup(self._db, self.DATA_SOURCE_NAME)
 
     def items_that_need_coverage(self, identifiers=None, **kwargs):
+        equivalent = aliased(Identifier)
```

Comment:
**tl;dr: I'd appreciate thoughts on reducing duplicates from the content db.**
This work with equivalencies (in addition to the removal of suppressed/superceded LicensePools) has all been added to try to reduce sending duplicate content items or Works to Bibblio. I don't have a lot of information about the duplicates through Bibblio, so I'm not sure whether they're different Works that represent the same book or different Editions/Representations that are the same Work.

Basically I don't know if this solution has any merit while adding an additional _db query to an already slow script. I think it might actually be better to do this equivalency comparison at the point that Representations are being selected, but that would make the whole thing take even *longer*, and I don't think it's worth it.